third_party/coreclr/include/coreclr/coreclr_delegates.h

```diff
@@ -51,9 +51,9 @@ typedef int (CORECLR_DELEGATE_CALLTYPE *load_assembly_fn)(
 
 typedef int (CORECLR_DELEGATE_CALLTYPE *load_assembly_bytes_fn)(
     const void *assembly_bytes      /* Bytes of the assembly to load */,
-    size_t     assembly_bytes_len   /* Byte length of the assembly to load */,
+    unsigned int     assembly_bytes_len   /* Byte length of the assembly to load */,
```

Comment:
Considering that it can change between platforms, there's a reason why it's a size_t (sizeof (size_t) == sizeof (void*))

Reply:
True, I'll restore this to a size_t and use cstdef.h instead